src/components/sendV2/summary/summary.js

```diff
@@ -144,16 +144,17 @@ class Summary extends React.Component {
   }
 
   render() {
+    const { account } = this.props;
     const { secondPassphrase, isHardwareWalletConnected } = this.state;
     let isBtnDisabled = secondPassphrase.hasSecondPassphrase && secondPassphrase.isValid !== '' && !secondPassphrase.isValid;
     isBtnDisabled = !isBtnDisabled && isHardwareWalletConnected;
 
     const confirmBtnMessage = isHardwareWalletConnected
-      ? this.props.t('Confirm on Ledger')
+      ? this.props.t(`Confirm on ${account.hwInfo.deviceModel}`)
```

Comment:
For the translations to work properly it should be
```suggestion
      ? this.props.t('Confirm on {{deviceModel}}', { deviceModel: account.hwInfo.deviceModel })
```

Reply:
fixed